index.html

```diff
@@ -200,59 +259,69 @@ <h1>About me</h1>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+              </ul>
+            </div>
+          </li>
 
-        <li>
-          <h3>
-            <div class="frameworks">&#9632;</div>Frameworks
-          </h3>
-          <ul>
-            <li>React.js</li>
-            <li>Ruby on Rails</li>
-            <li>RSpec</li>
-            <li>Capybera</li>
-          </ul>
-        </li>
+          <li>
+            <div>
+              <h3>
+                <div class="frameworks">&#9632;</div>Frameworks
+              </h3>
+              <ul>
+                <li>React.js</li>
+                <li>Ruby on Rails</li>
+                <li>RSpec</li>
+                <li>Capybera</li>
+              </ul>
+            </div>
+          </li>
 
-        <li>
-          <h3 class="sk">
-            <div class="skills">&#9679;</div>Skills
-          </h3>
-          <ul>
-            <li>Database Management</li>
-            <li>Version Control</li>
-            <li>CLI</li>
-            <li>API Design</li>
-            <li>Web Development</li>
-          </ul>
-        </li>
-      </ul>
+          <li>
+            <div>
+              <h3 class="sk">
+                <div class="skills">&#9679;</div>Skills
+              </h3>
+              <ul>
+                <li>Database Management</li>
+                <li>Version Control</li>
+                <li>CLI</li>
+                <li>API Design</li>
+                <li>Web Development</li>
+              </ul>
+            </div>
+          </li>
+        </ul>
+      </div>
     </section>
     <section class="contact">
-      <div class="announcement">
-        <p>I'm alway interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-      </div>
-      <form action="https://formspree.io/f/mqkorawk" method="POST" class="form">
-        <input type="text" placeholder="Full name" maxlength="30" required>
-        <input type="email" placeholder="Email address" required>
-        <textarea name="message" id="message" cols="30" rows="5" placeholder="Write me something..." maxlength="500"
-          required></textarea>
-        <div class="button">
-          <button type="submit">Get in touch</button>
+      <div class="wrapper">
+        <div class="announcement">
+          <p>I'm alway interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
         </div>
-      </form>
+        <form action="https://formspree.io/f/mqkorawk" method="POST" class="form">
+          <input type="text" placeholder="Full name" maxlength="30" required>
+          <input type="email" placeholder="Email address" required>
+          <textarea name="message" id="message" cols="30" rows="5" placeholder="Write me something..." maxlength="500"
+            required></textarea>
+          <div class="button">
+            <button type="submit">Get in touch</button>
+          </div>
```

Comment:
- [ ] I noticed a minor inconsistency in the `padding` of your contact section. In comparison to the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471), the padding appears slightly larger. It would be great if **you could adjust it to be closer to the provided design, ensuring greater consistency throughout your portfolio.** 🙌

### Your design:
![image](https://github.com/wacoo/Portfolio-Project/assets/107952724/f66cb5c7-941f-4fdc-a7ba-c3735e127a40)


### Figma design:
![image](https://github.com/wacoo/Portfolio-Project/assets/107952724/e3eefd5f-4784-44fe-8510-aab959e0554f)